pr_agent/git_providers/github_provider.py

```diff
@@ -178,7 +176,6 @@ def publish_code_suggestions(self, code_suggestions: list):
             if relevant_lines_end > relevant_lines_start:
                 post_parameters = {
                     "body": body,
-                    "commit_id": self.last_commit_id._identity,
                     "path": relevant_file,
                     "line": relevant_lines_end,
                     "start_line": relevant_lines_start,
```

Comment:
**Suggestion:** Consider adding type hints to the `post_parameters` dictionary for better readability and maintainability of the code.
```suggestion
                post_parameters: Dict[str, Union[str, int]] = {
                    "body": body,
                    "path": relevant_file,
                    "line": relevant_lines_end,
                    "start_line": relevant_lines_start,
                }
```